src/birdie/internal/titles.gleam

```diff
@@ -238,29 +238,32 @@ fn snap_call(
     glance.Call(
       function:,
       arguments: [
-        glance.Field(None, title),
-        glance.Field(Some("content"), _snapshot_content),
+        glance.UnlabelledField(title),
+        glance.LabelledField("content", _snapshot_content),
       ],
     )
     | glance.Call(
         function:,
         arguments: [
-          glance.Field(None, _snapshot_content),
-          glance.Field(_, title),
+          glance.UnlabelledField(_snapshot_content),
+          glance.LabelledField(_, title),
```

Comment:
This is different than what was here before: instead of an unlabelled and "any field" you're now matching on an unlabelled and a labelled one

Reply:
This now returns `Ok(acc)` instead of panicing

Reply:
This now also matches an UnlabelledField, i'm unsure wether a shorthand can be done here